src/util.rs

```diff
@@ -59,18 +59,31 @@ fn test_parse_size() {
     }
 }
 
-pub fn read_passphrase() -> io::Result<String> {
+fn read_passphrase_env() -> Option<String> {
```

Comment:
It seems to me this should return `io::Result<Option<String>>`. In case of an error, we should just return an error and handle it with a normal flow.

Reply:
Updated.